### PR TITLE
Binyli/bug fix

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

File: src/webportal/src/app/job-submission/components/docker-section.jsx [hunk 1]
```diff
@@ -187,14 +187,14 @@ export const DockerSection = ({onValueChange, value}) => {
     if (optionKey !== 'customize-image' && isCutomizedImageEnabled) {
       setCutomizedImageEnabled(false);
     }
-  }, []);
+  }, [uri]);
```
Comment (sunqinzheng) [inline src/webportal/src/app/job-submission/components/docker-section.jsx]: I think the `isCustomizedImageEnabled` state should be determined by 'uri' props.
You can use a `useMemo` hook to save the value.

Comment (Binyang2014) [reply src/webportal/src/app/job-submission/components/docker-section.jsx]: `isCustomizedImageEnabled ` is not just depends on the `uri` pop. Customers can also change its' value. So this situation could happens: `uri` not changed but `isCustomizedImageEnabled ` is changed by customer.
Maybe use state is better for this

Comment (sunqinzheng) [reply src/webportal/src/app/job-submission/components/docker-section.jsx]: `useEffect` will be triggered each time page is rerendered. So if user input an existing docker option in the custom docker url text field like `ufoym/deepo:tensorflow-py36-cu100`, this hook will be triggered and `isCutomizedImageEnabled` will be set to `false`.

**Comment by sunqinzheng** [comment]
By the way, the `ufoym/deepo:all` key in previous mapping function `getDockerImageOptionKey` is not matched with `options` object.

**Comment by Binyang2014** [comment]
> By the way, the `ufoym/deepo:all` key in previous mapping function `getDockerImageOptionKey` is not matched with `options` object.

Thanks for this, notice that

**Pull request merged**